app/models/group_invite_status.rb

```diff
@@ -3,6 +3,7 @@
 class GroupInviteStatus < ApplicationRecord
   ERRORED_STATUSES = %w[errored_creating_repo errored_importing_starter_code].freeze
   SETUP_STATUSES = %w[accepted waiting creating_repo importing_starter_code].freeze
+  LOCKED_STATUSES = %w[waiting creating_repo importing_starter_code].freeze
```

Comment:
Do you need this on individual invites as well? Can it be shared across both?

Reply:
They already exist on the individual `InviteStatus` model:
https://github.com/education/classroom/blob/master/app/models/invite_status.rb#L6

I will look into a clean way for the two models to share these constants.